src/gui/qgisinterface.h

```diff
@@ -947,6 +947,12 @@ class GUI_EXPORT QgisInterface : public QObject
     //! Remove action from the Web menu
     virtual void removePluginWebMenu( const QString &name, QAction *action ) = 0;
 
+    //! Add action to the Mesh menu
```

Comment:
Can you add a `\since QGIS 3.24` here?

I think it's fine to merge this still

Reply:
Done.